### ISSUE TITLE
Installation on OSX fails unless security settings are set to "allow apps downloaded from anywhere"

### ISSUE THREAD

**Issue by timlinux** [lektor/lektor#115]
# Problem

I just downloaded Lektor to try it out for the first time on OSX. After launching I get the typical instruction telling me that I need to set my security settings to allow this third party app. However after allowing it, I just get bounced back to the security settings message in an endless loop:

<img width="673" alt="screen shot 2016-01-17 at 20 41 10" src="https://cloud.githubusercontent.com/assets/178003/12379554/adbfa7b8-bd65-11e5-889c-64a84e3975c2.png">
<img width="667" alt="screen shot 2016-01-17 at 22 00 13" src="https://cloud.githubusercontent.com/assets/178003/12379556/b66061c8-bd65-11e5-84c5-e4105c53be48.png">

The only way I could resolve it is by setting my security settings to "allow apps downloaded from anywhere" instead of my preferred "From Mac App Store and preferred developers."
# Expected behaviour
- user should be able to keep their security settings on "From Mac App Store and preferred developers."
- after clicking 'allow' to create an exception the application should open instead of reverting back to the security preferences panel.

**Comment by fschulze** [comment]
I didn't have to change that setting. The regular way to solve this is to right click on the app and click "Open", you then get a dialog with an additional option to open the app anyway and it remembers that decision.

Ideally the releases would be signed to avoid this altogether though.

**Comment by mitsuhiko** [comment]
The releases are signed but a apparently the signature is broken.

**Comment by mitsuhiko** [comment]
Can someone check if 1.2 works for them?

**Comment by timlinux** [comment]
I'll grab a copy and test - thanks for following up on this!

**Comment by timlinux** [comment]
Just a follow up to say that I tested again with 1.2 and the problem still persists. Note I tested with some other application bundles and didn't experience the same issue.

**Comment by mitsuhiko** [comment]
@timlinux wtf. I don't understand any of this.

**Comment by mitsuhiko** [comment]
```
$ spctl --assess -v Lektor.app
Lektor.app: accepted
source=Developer ID

$ codesign -dv Lektor.app
Executable=/Users/mitsuhiko/Development/lektor/gui/build/Lektor-darwin-x64/Lektor.app/Contents/MacOS/Electron
Identifier=org.pocoo.lektor
Format=bundle with Mach-O thin (x86_64)
CodeDirectory v=20200 size=200 flags=0x0(none) hashes=3+3 location=embedded
Signature size=8562
Timestamp=01 Feb 2016 20:26:21
Info.plist entries=18
TeamIdentifier=ZRL6J4P42H
Sealed Resources version=2 rules=12 files=64
Internal requirements count=1 size=176
```

How does any of that make sense?

**Comment by mitsuhiko** [comment]
Notes for later: http://stackoverflow.com/questions/31709087/failure-digitally-signing-a-mac-app-outside-xcode

**Comment by mitsuhiko** [comment]
I'm out of ideas. I need help on this issue.

**Comment by mitsuhiko** [comment]
I think i fixed this in 1.2.1 now.

**Comment by timlinux** [comment]
Cool - I will download and test for you. Thanks!
